relayer-cli/src/commands/update.rs

```diff
@@ -0,0 +1,16 @@
+//! `update` subcommand
+
+use abscissa_core::{Command, Help, Options, Runnable};
+
+use crate::commands::tx::client::TxUpdateClientCmd;
+
+#[derive(Command, Debug, Options, Runnable)]
+pub enum UpdateCmds {
+    /// Generic `help`
+    #[options(help = "Get usage information")]
+    Help(Help<Self>),
+
+    /// Subcommand for updating a `client`
+    #[options(help = "Update an IBC client")]
+    Client(TxUpdateClientCmd),
+}
```

Comment:
Since we know the client's chain once it is created, maybe make this one `hermes update client ibc-0 07-tendermint-0` ( instead of `hermes update client ibc-0 ibc-1 07-tendermint-0`). Can do in a different PR also.

Reply:
I wanted to minimize the changes so that it's easy to get the PR reviewed ASAP. We should refactor some of the stuff in these files anyway, so we'll tackle your suggestion alongside further work (described in #773).